### ISSUE TITLE
Create a Pimcore backend configuration

### ISSUE THREAD

**Issue by rafaeloaguirre** [tksventures/generator-vue-storefront#5]
**Is your feature request related to a problem? Please describe.**
As it is right now, the generator produces the storefront and api with generic data retrieved from elasticsearch without using an eCommerce backend. 

Pimcore is one of the frameworks supported by Divante through https://github.com/DivanteLtd/coreshop-vsbridge/ and the one we are more familiar with. We need to allow for a generated storefront app to connect to a pimcore instance out of the box.

**Describe the solution you'd like**
Using the previously referenced link as a guide, we'll modify the config generation scripts to apply the pimcore instance's endpoint to the relevant configuration sections. This application would be triggered by the existence of a PIMCORE_ENDPOINT env variable.

**Comment by rafaeloaguirre** [comment]
Per discussion with the development team, we are switching our integration focus from Pimcore to Magento due to higher support for the latter framework.  We would follow very much the same logic in that the generation script would provide a docker-compose file with magento as a service along with its dependencies, default data for the store would not be pushed, and the generated vue-storefront app would contain the required changes needed to operate with magento out of the box.

**Comment by michalmikolajczyk** [comment]
nope, we don't need a generator script. Magento is released as as a Helm chart. We should only allow for an integration between Vue Storefront and Magento.
